terraform/modules/document-variations/provider.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_providers {
+    genesyscloud = {
+      source = "genesys.com/mypurecloud/genesyscloud"
+      version = "0.1.0"
+    }
+  }
+}
+
+# terraform {
```

Comment:
You have your provider pointing to a locally built version of a provider.  Someone downloading this blueprint will fail because they wont be able to locate the provider.

Reply:
@carnellj-genesys I'm waiting for the next release of the provider to go out to update the provider blocks.  The current version doesn't have all the resources needed yet, so I just left these and added the "do not merge" label for the time being